views/components/Drawer.jsx

```diff
@@ -176,56 +176,56 @@ class DrawerComponent extends Component {
         </div>
         <Divider />
         <List dense={true}>
-          <ListItem button={true} component="a" href={routes.home}>
+          <ListItem button={true} component='a' href={routes.home}>
```

Comment:
I think we should update the prettier settings to use double quotes for JSX props.